custom/icds_reports/reports/fact_sheets.py

```diff
@@ -561,6 +561,15 @@ def new_table_config(self):
                     }
                 ]
             )
+        else:
+            fact_sheet_data_config[4]['sections'][0]['rows_config'].append(
```

Comment:
nit: I think it could be nice to extract `fact_sheet_data_config[4]['sections'][0]['rows_config']` to a well-named variable. lot of magic knowledge embedded there.